lock-manager/deployment/helm/templates/deployment.yaml

```diff
@@ -27,10 +27,10 @@ spec:
           resources:
             limits:
               cpu: "1"
-              memory: "128Mi"
+              memory: "256Mi"
```

Comment:
Started getting OOM on local, so bumped it up a little

Reply:
Although this is fine for now, if we get OOM again, we may have to profile and see why we get OOM. What surprises me though is why Lock manager would run into OOM for 128MB. Lets see how it goes with 256MB